esp/platform/sechandler.cpp

```diff
@@ -95,6 +95,8 @@ bool SecHandler::authorizeSecFeature(const char * pszFeatureUrl, const char* Use
     if(pSecondaryUser.get()== NULL)
     {
         pSecondaryUser.setown(m_secmgr->createUser(UserID));
+	if (!pSecondaryUser.get())
```

Comment:
minor: indentation (this line and next)
trivial: you do not need to call .get()